runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -87,6 +88,8 @@
 
 	private boolean paused = false;
 
+	private static final DecimalFormat twoDecimalPlaces = new DecimalFormat("0.00");
```

Comment:
should be named UPPER_CASE something like `TWO_DECIMAL_FORMAT`

Reply:
Whoops. Fixed.